### DISCUSSION TITLE
Does graphql-java support distributed schema architectures(federeation/schema-stitching)?

### DISCUSSION THREAD

**Discussion by deepakguruu** [graphql-java/graphql-java#2204] [Q&A] [answered]
I'm building a cloud based graphql architecture, so I wanted to know if distributed schema architectures were supported by graphql-java.

**Comment by bbakerman** [comment, marked as answer]
> wanted to know if distributed schema architectures were supported by graphql-java

No it does not.

graphql-java is a base library for executing graphql queries.  It deliberately tries to be a raw graphql engine and not an application framework say.

However there are JVM federation projects out there and all of them use graphql-java under the covers.

https://github.com/apollographql/federation-jvm

https://github.com/Netflix/dgs-framework

**Reply by deepakguruu** [reply]
Will look into those.
Thanks for the quick reply :)
